src/client/pkg/require/require.go

```diff
@@ -95,48 +95,234 @@ func NotEqual(tb testing.TB, expected interface{}, actual interface{}, msgAndArg
 	}
 }
 
-// EqualOneOf checks if a value is equal to one of the elements of a slice.
-func EqualOneOf(tb testing.TB, expecteds []interface{}, actual interface{}, msgAndArgs ...interface{}) {
+// elementsEqualOrErr is a helper for ElementsEqualOrError and ElementsEqual
+// that implements their essential logic (comparing the values of 'expecteds'
+// and 'actuals).  The 't' argument is optional (may be set to nil), so that
+// ElementsEqual can pass it, and its error's line number will be set
+// appropriately, while ElementsEqualOrError (which does not accept a 't'
+// argument' can pass nil)
+//
+// As described in the comments for 'ElementsEqual()', if the elements of
+// 'expecteds' and 'actuals' are pointers, elementsEqualOrErr will unwrap the
+// pointers before comparing them, and 'nil' and the empty slice are treated as
+// equivalent
+func elementsEqualOrErr(tb testing.TB, expecteds interface{}, actuals interface{}) error {
+	if tb != nil {
+		tb.Helper()
+	}
```

Comment:
Is this needed? It looks like all of the methods that call this call tb.Helper themselves before passing it in. And it doesn't look like there's any place where it's `nil`. I could just be missing them though.